cmd/init.go

```diff
@@ -92,12 +92,15 @@ dapr init -s
 			print.SuccessStatusEvent(os.Stdout, fmt.Sprintf("Success! Dapr has been installed to namespace %s. To verify, run `dapr status -k' in your terminal. To get started, go here: https://aka.ms/dapr-getting-started", config.Namespace))
 		} else {
 			dockerNetwork := ""
-			imageRepositoryURL := ""
+			imageRegistryURL := ""
 			if !slimMode {
 				dockerNetwork = viper.GetString("network")
-				imageRepositoryURL = viper.GetString("image-repository")
+				imageRegistryURL = viper.GetString("image-registry")
 			}
-			err := standalone.Init(runtimeVersion, dashboardVersion, dockerNetwork, slimMode, imageRepositoryURL)
+			if imageRegistryURL != "" {
+				print.InfoStatusEvent(os.Stdout, "Flag --image-registry is a preview feature and only available from cli version 1.7 onwards")
```

Comment:
```suggestion
				print.WarningStatusEvent(os.Stdout, "Flag --image-registry is a preview feature and is subject to change. It only available from CLI version 1.7 onwards.")
```

Reply:
Add it in docs https://github.com/dapr/docs/blob/v1.7/daprdocs/content/en/operations/support/support-preview-features.md.

Also add `unix-domain-sockets` to the list of preview features.